website/content/api-docs/secret/aws.mdx

```diff
@@ -46,9 +57,19 @@ valid AWS credentials with proper permissions.
   recoverable errors. The default (`-1`) falls back to the AWS SDK's default
   behavior.
 
-- `access_key` `(string: <required>)` – Specifies the AWS access key ID.
+- `access_key` `(string: "")` – Specifies the AWS access key ID. Mutually exclusive with
+  `identity_token_audience`.
+
+- `secret_key` `(string: "")` – Specifies the AWS secret access key. Must be provided with
+  `access_key`.
+
+- `role_arn` `(string: "")` – Role ARN to assume for plugin identity token federation.
+  Required with `identity_token_audience`.
+
+- `identity_token_audience` `(string: "")` - The audience claim value for plugin identity tokens.
+  Mutually exclusive with `access_key`.
 
-- `secret_key` `(string: <required>)` – Specifies the AWS secret access key.
+- `identity_token_ttl` `(string/int: 3600)` - The TTL of generated tokens. Defaults to 1 hour.
```

Comment:
Could also mention it can use duration string formats like "Uses [duration format strings](/vault/docs/concepts/duration-format).".

Reply:
Updated